components/api_server/src/model/issue_tracker.py

```diff
@@ -113,29 +145,39 @@ def create_issue(self, summary: str, description: str = "") -> tuple[str, str]:
         for attribute, name in [(self.url, "URL"), (project_key, "project key"), (issue_type, "issue type")]:
             if not attribute:
                 return "", f"Issue tracker has no {name} configured."
-        api_url = self.issue_creation_api % self.url
-        json = {
-            "fields": {
-                "project": {"key": project_key},
-                "issuetype": {"name": issue_type},
-                "summary": summary,
-                "description": description,
-            },
+        api_url = self.issue_creation_api
```

Comment:
Why still create this local var, instead of using `self.issue_creation_api` in the two cases below?